xsdvalidate/xsd_validate_activity.go

```diff
@@ -0,0 +1,192 @@
+package xsdvalidate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	xsdvalidator "github.com/terminalstatic/go-xsd-validate"
```

Comment:
It looks like https://github.com/terminalstatic/go-xsd-validate uses cgo to run the C libxml2 shared library. I've never used the cgo C linking functionality but I believe @sevein raised some concerns about it when discussing XSD validation with @jraddaoui, hopefully they will respond to this comment with more information about why we may not want to use C bindings.